playwright/login.ts

```diff
@@ -0,0 +1,11 @@
+import * as config from './config.json';
+
+export default async function login(page): Promise<void> {
+  await Promise.all([
+    await page.fill('input#account', config.accountLogin),
+    page.click('button#login-submit'),
```

Comment:
- missing await
- it's better to use locators instead of page.click directly

Reply:
Done ✅ Using `await page.$('button#login-submit');`